[Contributing Help] Failing Tests - normalize_drive() assertions

I followed the instructions [here](https://github.com/pypa/pipenv/blob/master/CONTRIBUTING.rst) to setup the development environment but I can't get the test suite to pass.

Not sure if it's an issue with the test suite or my dev environmnet - [pipenv.help output](https://github.com/pypa/pipenv/files/1971876/test-troubleshoot.txt)

Steps by step and test suite tracebacks are shown below.

# Setup Venv
``` 
D:\Dropbox\shared\dev\repos
λ cd pipenv

D:\Dropbox\shared\dev\repos\pipenv (master -> origin)
λ virtualenv %userprofile%/.env/pipenv-dev-venv

Using base prefix 'c:\\python36'
New python executable in C:\Users\gtalarico\.env\pipenv-dev-venv\Scripts\python.exe
Installing setuptools, pip, wheel...done.

D:\Dropbox\shared\dev\repos\pipenv (master -> origin)
λ %userprofile%\.env\pipenv-dev-venv\Scripts\activate.bat

D:\Dropbox\shared\dev\repos\pipenv (master -> origin)
(pipenv-dev-venv) λ which python

/c/Users/gtalarico/.env/pipenv-dev-venv/Scripts/python

```

# Setup.py
```

D:\Dropbox\shared\dev\repos\pipenv (master -> origin)
(pipenv-dev-venv) λ python setup.py develop

[ ... ] Installed all successfully

Finished processing dependencies for pipenv==11.10.1

```

# Install dev dependencies
```
D:\Dropbox\shared\dev\repos\pipenv (master -> origin)
(pipenv-dev-venv) λ pipenv install --dev

Courtesy Notice: Pipenv found itself running within a virtual environment, so it will automatically use that environment, instead of creating its own for any project. You can set PIPENV_IGNORE_VIRTUALENVS=1 to force pipenv to ignore that environment and create its own instead.
Installing dependencies from Pipfile.lock (ae475c)…
Ignoring configparser: markers 'python_version < "3.2"' don't match your environment
Ignoring enum34: markers 'python_version < "3.4"' don't match your environment
Ignoring funcsigs: markers 'python_version < "3.0"' don't match your environment
Ignoring stdeb: markers 'sys_platform == "linux"' don't match your environment
  ================================ 55/55 - 00:00:40
An error occurred while installing -e .! Will try again.
Installing initially–failed dependencies…
Success installing -e .!------------- 0/1 - 00:00:00
     ================================ 1/1 - 00:00:03

```

# Run Tests (Passing 163 of 168)

5 Are failing on the same assertion:

```
> assert normalize_drive(p.path) in p.pipenv('--venv').out
E               AssertionError: assert 'C:\\Users\\GTALAR~1\\AppData\\Local\\Temp\\pipenv-mu0ipp4h-project' in 'C:\\Users\\gtalarico\\AppData\\Local\\Temp\\pipenv-mu0ipp4h-project\\.venv\n'
```

# Full Test Results Traceback

```
D:\Dropbox\shared\dev\repos\pipenv (master -> origin)
(pipenv-dev-venv) λ pipenv run pytest tests --lf

Courtesy Notice: Pipenv found itself running within a virtual environment, so it will automatically use that environment, instead of creating its own for any project. You can set PIPENV_IGNORE_VIRTUALENVS=1 to force pipenv to ignore that environment and create its own instead.
Loading .env environment variables…
============================= test session starts =============================
platform win32 -- Python 3.6.3, pytest-3.5.1, py-1.5.3, pluggy-0.6.0
rootdir: D:\Dropbox\shared\dev\repos\pipenv, inifile:
plugins: xdist-1.22.2, tap-2.2, forked-0.2, flaky-3.4.0, pypi-0.1.1
collected 168 items / 163 deselected
run-last-failure: rerun previous 5 failures

tests\integration\test_cli.py F                                          [ 20%]
tests\integration\test_dot_venv.py FFF                                   [ 80%]
tests\integration\test_windows.py F                                      [100%]

================================== FAILURES ===================================
______________________________ test_pipenv_where ______________________________

PipenvInstance = <class 'tests.integration.conftest._PipenvInstance'>
pypi_secure = <pytest_pypi.serve.SecureServer object at 0x0000028F73AD4160>

    @pytest.mark.cli
    def test_pipenv_where(PipenvInstance, pypi_secure):
        with PipenvInstance(pypi=pypi_secure) as p:
>           assert normalize_drive(p.path) in p.pipenv('--where').out
E           AssertionError: assert 'C:\\Users\\GTALAR~1\\AppData\\Local\\Temp\\pipenv-m669d1c3-project' in 'C:\\Users\\gtalarico\\AppData\\Local\\Temp\\pipenv-m669d1c3-project\n'
E            +  where 'C:\\Users\\GTALAR~1\\AppData\\Local\\Temp\\pipenv-m669d1c3-project' = normalize_drive('C:\\Users\\GTALAR~1\\AppData\\Local\\Temp\\pipenv-m669d1c3-project')
E            +    where 'C:\\Users\\GTALAR~1\\AppData\\Local\\Temp\\pipenv-m669d1c3-project' = <tests.integration.conftest._PipenvInstance object at 0x0000028F73AD4BE0>.path
E            +  and   'C:\\Users\\gtalarico\\AppData\\Local\\Temp\\pipenv-m669d1c3-project\n' = <Command 'pipenv --where'>.out
E            +    where <Command 'pipenv --where'> = <bound method _PipenvInstance.pipenv of <tests.integration.conftest._PipenvInstance object at 0x0000028F73AD4BE0>>('--where')
E            +      where <bound method _PipenvInstance.pipenv of <tests.integration.conftest._PipenvInstance object at 0x0000028F73AD4BE0>> = <tests.integration.conftest._PipenvInstance object at 0x0000028F73AD4BE0>.pipenv

tests\integration\test_cli.py:15: AssertionError
---------------------------- Captured stdout call -----------------------------
$ pipenv --where
C:\Users\gtalarico\AppData\Local\Temp\pipenv-m669d1c3-project


____________________________ test_venv_in_project _____________________________

PipenvInstance = <class 'tests.integration.conftest._PipenvInstance'>
pypi = <pytest_pypi.serve.Server object at 0x0000028F73D11208>

    @pytest.mark.dotvenv
    def test_venv_in_project(PipenvInstance, pypi):
        with temp_environ():
            os.environ['PIPENV_VENV_IN_PROJECT'] = '1'
            with PipenvInstance(pypi=pypi) as p:
                c = p.pipenv('install requests')
                assert c.return_code == 0
>               assert normalize_drive(p.path) in p.pipenv('--venv').out
E               AssertionError: assert 'C:\\Users\\GTALAR~1\\AppData\\Local\\Temp\\pipenv-ey6v2ivk-project' in 'C:\\Users\\gtalarico\\AppData\\Local\\Temp\\pipenv-ey6v2ivk-project\\.venv\n'
E                +  where 'C:\\Users\\GTALAR~1\\AppData\\Local\\Temp\\pipenv-ey6v2ivk-project' = normalize_drive('C:\\Users\\GTALAR~1\\AppData\\Local\\Temp\\pipenv-ey6v2ivk-project')
E                +    where 'C:\\Users\\GTALAR~1\\AppData\\Local\\Temp\\pipenv-ey6v2ivk-project' = <tests.integration.conftest._PipenvInstance object at 0x0000028F73D11358>.path
E                +  and   'C:\\Users\\gtalarico\\AppData\\Local\\Temp\\pipenv-ey6v2ivk-project\\.venv\n' = <Command 'pipenv --venv'>.out
E                +    where <Command 'pipenv --venv'> = <bound method _PipenvInstance.pipenv of <tests.integration.conftest._PipenvInstance object at 0x0000028F73D11358>>('--venv')
E                +      where <bound method _PipenvInstance.pipenv of <tests.integration.conftest._PipenvInstance object at 0x0000028F73D11358>> = <tests.integration.conftest._PipenvInstance object at 0x0000028F73D11358>.pipenv

tests\integration\test_dot_venv.py:17: AssertionError
---------------------------- Captured stdout call -----------------------------
$ pipenv install requests
Installing requestsâ€¦
Looking in indexes: http://127.0.0.1:55733/simple
Collecting requests
  Downloading http://127.0.0.1:55733/requests/requests-2.18.4-py2.py3-none-any.whl (88kB)
Collecting urllib3<1.23,>=1.21.1 (from requests)
  Downloading http://127.0.0.1:55733/urllib3/urllib3-1.22-py2.py3-none-any.whl (132kB)
Collecting idna<2.7,>=2.5 (from requests)
  Downloading http://127.0.0.1:55733/idna/idna-2.6-py2.py3-none-any.whl (56kB)
Collecting chardet<3.1.0,>=3.0.2 (from requests)
  Downloading http://127.0.0.1:55733/chardet/chardet-3.0.4-py2.py3-none-any.whl (133kB)
Collecting certifi>=2017.4.17 (from requests)
  Downloading http://127.0.0.1:55733/certifi/certifi-2018.1.18-py2.py3-none-any.whl (151kB)
Installing collected packages: urllib3, idna, chardet, certifi, requests
Successfully installed certifi-2018.1.18 chardet-3.0.4 idna-2.6 requests-2.18.4 urllib3-1.22

Adding requests to Pipfile's [packages]â€¦
Installing dependencies from Pipfile.lock (3c0fff)â€¦
To activate this project's virtualenv, run the following:
 $ pipenv shell

Creating a virtualenv for this projectâ€¦
Using C:\Users\gtalarico\.env\pipenv-dev-venv\Scripts\python.exe (3.6.3) to create virtualenvâ€¦
Already using interpreter C:\Users\gtalarico\.env\pipenv-dev-venv\Scripts\python.exe

Using real prefix 'c:\\python36'

New python executable in C:\Users\gtalarico\AppData\Local\Temp\pipenv-ey6v2ivk-project\.venv\Scripts\python.exe

Installing setuptools, pip, wheel...done.


Virtualenv location: C:\Users\gtalarico\AppData\Local\Temp\pipenv-ey6v2ivk-project\.venv
Creating a Pipfile for this projectâ€¦
Pipfile.lock not found, creatingâ€¦
Locking [dev-packages] dependenciesâ€¦
Locking [packages] dependenciesâ€¦
Updated Pipfile.lock (3c0fff)!

$ pipenv --venv
C:\Users\gtalarico\AppData\Local\Temp\pipenv-ey6v2ivk-project\.venv


---------------------------- Captured stderr call -----------------------------
127.0.0.1 - - [03/May/2018 12:43:51] "GET /simple/requests/ HTTP/1.1" 200 669
127.0.0.1 - - [03/May/2018 12:43:51] "GET /requests/requests-2.18.4-py2.py3-none-any.whl HTTP/1.1" 200 88704
127.0.0.1 - - [03/May/2018 12:43:51] "GET /simple/urllib3/ HTTP/1.1" 200 352
127.0.0.1 - - [03/May/2018 12:43:51] "GET /urllib3/urllib3-1.22-py2.py3-none-any.whl HTTP/1.1" 200 132332
127.0.0.1 - - [03/May/2018 12:43:51] "GET /simple/idna/ HTTP/1.1" 200 318
127.0.0.1 - - [03/May/2018 12:43:51] "GET /idna/idna-2.6-py2.py3-none-any.whl HTTP/1.1" 200 56450
127.0.0.1 - - [03/May/2018 12:43:51] "GET /simple/chardet/ HTTP/1.1" 200 356
127.0.0.1 - - [03/May/2018 12:43:51] "GET /chardet/chardet-3.0.4-py2.py3-none-any.whl HTTP/1.1" 200 133356
127.0.0.1 - - [03/May/2018 12:43:51] "GET /simple/certifi/ HTTP/1.1" 200 372
127.0.0.1 - - [03/May/2018 12:43:51] "GET /certifi/certifi-2018.1.18-py2.py3-none-any.whl HTTP/1.1" 200 151607
127.0.0.1 - - [03/May/2018 12:43:55] "GET /simple/requests/ HTTP/1.1" 200 669
127.0.0.1 - - [03/May/2018 12:43:55] "GET /simple/requests/ HTTP/1.1" 200 669
127.0.0.1 - - [03/May/2018 12:43:55] "GET /requests/requests-2.18.4-py2.py3-none-any.whl HTTP/1.1" 200 88704
127.0.0.1 - - [03/May/2018 12:43:55] "GET /simple/certifi/ HTTP/1.1" 200 372
127.0.0.1 - - [03/May/2018 12:43:55] "GET /simple/chardet/ HTTP/1.1" 200 356
127.0.0.1 - - [03/May/2018 12:43:55] "GET /simple/idna/ HTTP/1.1" 200 318
127.0.0.1 - - [03/May/2018 12:43:55] "GET /simple/urllib3/ HTTP/1.1" 200 352
127.0.0.1 - - [03/May/2018 12:43:55] "GET /simple/urllib3/ HTTP/1.1" 200 352
127.0.0.1 - - [03/May/2018 12:43:55] "GET /urllib3/urllib3-1.22-py2.py3-none-any.whl HTTP/1.1" 200 132332
127.0.0.1 - - [03/May/2018 12:43:55] "GET /simple/idna/ HTTP/1.1" 200 318
127.0.0.1 - - [03/May/2018 12:43:55] "GET /idna/idna-2.6-py2.py3-none-any.whl HTTP/1.1" 200 56450
127.0.0.1 - - [03/May/2018 12:43:55] "GET /simple/certifi/ HTTP/1.1" 200 372
127.0.0.1 - - [03/May/2018 12:43:55] "GET /certifi/certifi-2018.1.18-py2.py3-none-any.whl HTTP/1.1" 200 151607
127.0.0.1 - - [03/May/2018 12:43:55] "GET /simple/chardet/ HTTP/1.1" 200 356
127.0.0.1 - - [03/May/2018 12:43:55] "GET /chardet/chardet-3.0.4-py2.py3-none-any.whl HTTP/1.1" 200 133356
127.0.0.1 - - [03/May/2018 12:43:56] "GET /simple/chardet/ HTTP/1.1" 200 356
127.0.0.1 - - [03/May/2018 12:43:56] "GET /chardet/chardet-3.0.4.tar.gz HTTP/1.1" 200 1868453
127.0.0.1 - - [03/May/2018 12:43:56] "GET /chardet/chardet-3.0.4-py2.py3-none-any.whl HTTP/1.1" 200 133356
127.0.0.1 - - [03/May/2018 12:43:56] "GET /simple/requests/ HTTP/1.1" 200 669
127.0.0.1 - - [03/May/2018 12:43:56] "GET /requests/requests-2.18.4.tar.gz HTTP/1.1" 200 126224
127.0.0.1 - - [03/May/2018 12:43:56] "GET /requests/requests-2.18.4-py2.py3-none-any.whl HTTP/1.1" 200 88704
127.0.0.1 - - [03/May/2018 12:43:56] "GET /simple/urllib3/ HTTP/1.1" 200 352
127.0.0.1 - - [03/May/2018 12:43:56] "GET /urllib3/urllib3-1.22.tar.gz HTTP/1.1" 200 226083
127.0.0.1 - - [03/May/2018 12:43:56] "GET /urllib3/urllib3-1.22-py2.py3-none-any.whl HTTP/1.1" 200 132332
127.0.0.1 - - [03/May/2018 12:43:56] "GET /simple/certifi/ HTTP/1.1" 200 372
127.0.0.1 - - [03/May/2018 12:43:56] "GET /certifi/certifi-2018.1.18.tar.gz HTTP/1.1" 200 151258
127.0.0.1 - - [03/May/2018 12:43:56] "GET /certifi/certifi-2018.1.18-py2.py3-none-any.whl HTTP/1.1" 200 151607
127.0.0.1 - - [03/May/2018 12:43:56] "GET /simple/idna/ HTTP/1.1" 200 318
127.0.0.1 - - [03/May/2018 12:43:56] "GET /idna/idna-2.6-py2.py3-none-any.whl HTTP/1.1" 200 56450
127.0.0.1 - - [03/May/2018 12:43:56] "GET /idna/idna-2.6.tar.gz HTTP/1.1" 200 135992
__________________________ test_venv_at_project_root __________________________

PipenvInstance = <class 'tests.integration.conftest._PipenvInstance'>

    @pytest.mark.dotvenv
    def test_venv_at_project_root(PipenvInstance):
        with temp_environ():
            with PipenvInstance(chdir=True) as p:
                os.environ['PIPENV_VENV_IN_PROJECT'] = '1'
                c = p.pipenv('install')
                assert c.return_code == 0
>               assert normalize_drive(p.path) in p.pipenv('--venv').out
E               AssertionError: assert 'C:\\Users\\GTALAR~1\\AppData\\Local\\Temp\\pipenv-mu0ipp4h-project' in 'C:\\Users\\gtalarico\\AppData\\Local\\Temp\\pipenv-mu0ipp4h-project\\.venv\n'
E                +  where 'C:\\Users\\GTALAR~1\\AppData\\Local\\Temp\\pipenv-mu0ipp4h-project' = normalize_drive('C:\\Users\\GTALAR~1\\AppData\\Local\\Temp\\pipenv-mu0ipp4h-project')
E                +    where 'C:\\Users\\GTALAR~1\\AppData\\Local\\Temp\\pipenv-mu0ipp4h-project' = <tests.integration.conftest._PipenvInstance object at 0x0000028F73D48550>.path
E                +  and   'C:\\Users\\gtalarico\\AppData\\Local\\Temp\\pipenv-mu0ipp4h-project\\.venv\n' = <Command 'pipenv --venv'>.out
E                +    where <Command 'pipenv --venv'> = <bound method _PipenvInstance.pipenv of <tests.integration.conftest._PipenvInstance object at 0x0000028F73D48550>>('--venv')
E                +      where <bound method _PipenvInstance.pipenv of <tests.integration.conftest._PipenvInstance object at 0x0000028F73D48550>> = <tests.integration.conftest._PipenvInstance object at 0x0000028F73D48550>.pipenv

tests\integration\test_dot_venv.py:27: AssertionError
---------------------------- Captured stdout call -----------------------------
$ pipenv install
Installing dependencies from Pipfile.lock (742473)â€¦
To activate this project's virtualenv, run the following:
 $ pipenv shell

Creating a virtualenv for this projectâ€¦
Using C:\Users\gtalarico\.env\pipenv-dev-venv\Scripts\python.exe (3.6.3) to create virtualenvâ€¦
Already using interpreter C:\Users\gtalarico\.env\pipenv-dev-venv\Scripts\python.exe

Using real prefix 'c:\\python36'

New python executable in C:\Users\gtalarico\AppData\Local\Temp\pipenv-mu0ipp4h-project\.venv\Scripts\python.exe

Installing setuptools, pip, wheel...done.


Virtualenv location: C:\Users\gtalarico\AppData\Local\Temp\pipenv-mu0ipp4h-project\.venv
Creating a Pipfile for this projectâ€¦
Pipfile.lock not found, creatingâ€¦
Locking [dev-packages] dependenciesâ€¦
Locking [packages] dependenciesâ€¦
Updated Pipfile.lock (742473)!

$ pipenv --venv
C:\Users\gtalarico\AppData\Local\Temp\pipenv-mu0ipp4h-project\.venv


__________________________ test_reuse_previous_venv ___________________________

PipenvInstance = <class 'tests.integration.conftest._PipenvInstance'>
pypi = <pytest_pypi.serve.Server object at 0x0000028F73D11208>

    @pytest.mark.dotvenv
    def test_reuse_previous_venv(PipenvInstance, pypi):
        with PipenvInstance(chdir=True, pypi=pypi) as p:
            os.mkdir('.venv')
            c = p.pipenv('install requests')
            assert c.return_code == 0
>           assert normalize_drive(p.path) in p.pipenv('--venv').out
E           AssertionError: assert 'C:\\Users\\GTALAR~1\\AppData\\Local\\Temp\\pipenv-_k5luah7-project' in 'C:\\Users\\gtalarico\\AppData\\Local\\Temp\\pipenv-_k5luah7-project\\.venv\n'
E            +  where 'C:\\Users\\GTALAR~1\\AppData\\Local\\Temp\\pipenv-_k5luah7-project' = normalize_drive('C:\\Users\\GTALAR~1\\AppData\\Local\\Temp\\pipenv-_k5luah7-project')
E            +    where 'C:\\Users\\GTALAR~1\\AppData\\Local\\Temp\\pipenv-_k5luah7-project' = <tests.integration.conftest._PipenvInstance object at 0x0000028F73D57400>.path
E            +  and   'C:\\Users\\gtalarico\\AppData\\Local\\Temp\\pipenv-_k5luah7-project\\.venv\n' = <Command 'pipenv --venv'>.out
E            +    where <Command 'pipenv --venv'> = <bound method _PipenvInstance.pipenv of <tests.integration.conftest._PipenvInstance object at 0x0000028F73D57400>>('--venv')
E            +      where <bound method _PipenvInstance.pipenv of <tests.integration.conftest._PipenvInstance object at 0x0000028F73D57400>> = <tests.integration.conftest._PipenvInstance object at 0x0000028F73D57400>.pipenv

tests\integration\test_dot_venv.py:41: AssertionError
---------------------------- Captured stdout call -----------------------------
$ pipenv install requests
Installing requestsâ€¦
Looking in indexes: http://127.0.0.1:55733/simple
Collecting requests
  Downloading http://127.0.0.1:55733/requests/requests-2.18.4-py2.py3-none-any.whl (88kB)
Collecting chardet<3.1.0,>=3.0.2 (from requests)
  Downloading http://127.0.0.1:55733/chardet/chardet-3.0.4-py2.py3-none-any.whl (133kB)
Collecting idna<2.7,>=2.5 (from requests)
  Downloading http://127.0.0.1:55733/idna/idna-2.6-py2.py3-none-any.whl (56kB)
Collecting certifi>=2017.4.17 (from requests)
  Downloading http://127.0.0.1:55733/certifi/certifi-2018.1.18-py2.py3-none-any.whl (151kB)
Collecting urllib3<1.23,>=1.21.1 (from requests)
  Downloading http://127.0.0.1:55733/urllib3/urllib3-1.22-py2.py3-none-any.whl (132kB)
Installing collected packages: chardet, idna, certifi, urllib3, requests
Successfully installed certifi-2018.1.18 chardet-3.0.4 idna-2.6 requests-2.18.4 urllib3-1.22

Adding requests to Pipfile's [packages]â€¦
Installing dependencies from Pipfile.lock (3c0fff)â€¦
To activate this project's virtualenv, run the following:
 $ pipenv shell

Creating a virtualenv for this projectâ€¦
Using C:\Users\gtalarico\.env\pipenv-dev-venv\Scripts\python.exe (3.6.3) to create virtualenvâ€¦
Already using interpreter C:\Users\gtalarico\.env\pipenv-dev-venv\Scripts\python.exe

Using real prefix 'c:\\python36'

New python executable in C:\Users\gtalarico\AppData\Local\Temp\pipenv-_k5luah7-project\.venv\Scripts\python.exe

Installing setuptools, pip, wheel...done.


Virtualenv location: C:\Users\gtalarico\AppData\Local\Temp\pipenv-_k5luah7-project\.venv
Creating a Pipfile for this projectâ€¦
Pipfile.lock not found, creatingâ€¦
Locking [dev-packages] dependenciesâ€¦
Locking [packages] dependenciesâ€¦
Updated Pipfile.lock (3c0fff)!

$ pipenv --venv
C:\Users\gtalarico\AppData\Local\Temp\pipenv-_k5luah7-project\.venv


---------------------------- Captured stderr call -----------------------------
127.0.0.1 - - [03/May/2018 12:44:36] "GET /simple/requests/ HTTP/1.1" 200 669
127.0.0.1 - - [03/May/2018 12:44:36] "GET /requests/requests-2.18.4-py2.py3-none-any.whl HTTP/1.1" 200 88704
127.0.0.1 - - [03/May/2018 12:44:36] "GET /simple/chardet/ HTTP/1.1" 200 356
127.0.0.1 - - [03/May/2018 12:44:36] "GET /chardet/chardet-3.0.4-py2.py3-none-any.whl HTTP/1.1" 200 133356
127.0.0.1 - - [03/May/2018 12:44:37] "GET /simple/idna/ HTTP/1.1" 200 318
127.0.0.1 - - [03/May/2018 12:44:37] "GET /idna/idna-2.6-py2.py3-none-any.whl HTTP/1.1" 200 56450
127.0.0.1 - - [03/May/2018 12:44:37] "GET /simple/certifi/ HTTP/1.1" 200 372
127.0.0.1 - - [03/May/2018 12:44:37] "GET /certifi/certifi-2018.1.18-py2.py3-none-any.whl HTTP/1.1" 200 151607
127.0.0.1 - - [03/May/2018 12:44:37] "GET /simple/urllib3/ HTTP/1.1" 200 352
127.0.0.1 - - [03/May/2018 12:44:37] "GET /urllib3/urllib3-1.22-py2.py3-none-any.whl HTTP/1.1" 200 132332
127.0.0.1 - - [03/May/2018 12:44:41] "GET /simple/requests/ HTTP/1.1" 200 669
127.0.0.1 - - [03/May/2018 12:44:41] "GET /simple/requests/ HTTP/1.1" 200 669
127.0.0.1 - - [03/May/2018 12:44:41] "GET /requests/requests-2.18.4-py2.py3-none-any.whl HTTP/1.1" 200 88704
127.0.0.1 - - [03/May/2018 12:44:41] "GET /simple/certifi/ HTTP/1.1" 200 372
127.0.0.1 - - [03/May/2018 12:44:41] "GET /simple/chardet/ HTTP/1.1" 200 356
127.0.0.1 - - [03/May/2018 12:44:41] "GET /simple/idna/ HTTP/1.1" 200 318
127.0.0.1 - - [03/May/2018 12:44:41] "GET /simple/urllib3/ HTTP/1.1" 200 352
127.0.0.1 - - [03/May/2018 12:44:41] "GET /simple/idna/ HTTP/1.1" 200 318
127.0.0.1 - - [03/May/2018 12:44:41] "GET /idna/idna-2.6-py2.py3-none-any.whl HTTP/1.1" 200 56450
127.0.0.1 - - [03/May/2018 12:44:41] "GET /simple/certifi/ HTTP/1.1" 200 372
127.0.0.1 - - [03/May/2018 12:44:41] "GET /certifi/certifi-2018.1.18-py2.py3-none-any.whl HTTP/1.1" 200 151607
127.0.0.1 - - [03/May/2018 12:44:41] "GET /simple/urllib3/ HTTP/1.1" 200 352
127.0.0.1 - - [03/May/2018 12:44:41] "GET /urllib3/urllib3-1.22-py2.py3-none-any.whl HTTP/1.1" 200 132332
127.0.0.1 - - [03/May/2018 12:44:41] "GET /simple/chardet/ HTTP/1.1" 200 356
127.0.0.1 - - [03/May/2018 12:44:41] "GET /chardet/chardet-3.0.4-py2.py3-none-any.whl HTTP/1.1" 200 133356
127.0.0.1 - - [03/May/2018 12:44:41] "GET /simple/idna/ HTTP/1.1" 200 318
127.0.0.1 - - [03/May/2018 12:44:41] "GET /idna/idna-2.6.tar.gz HTTP/1.1" 200 135992
127.0.0.1 - - [03/May/2018 12:44:41] "GET /idna/idna-2.6-py2.py3-none-any.whl HTTP/1.1" 200 56450
127.0.0.1 - - [03/May/2018 12:44:41] "GET /simple/requests/ HTTP/1.1" 200 669
127.0.0.1 - - [03/May/2018 12:44:42] "GET /requests/requests-2.18.4-py2.py3-none-any.whl HTTP/1.1" 200 88704
127.0.0.1 - - [03/May/2018 12:44:42] "GET /requests/requests-2.18.4.tar.gz HTTP/1.1" 200 126224
127.0.0.1 - - [03/May/2018 12:44:42] "GET /simple/certifi/ HTTP/1.1" 200 372
127.0.0.1 - - [03/May/2018 12:44:42] "GET /certifi/certifi-2018.1.18.tar.gz HTTP/1.1" 200 151258
127.0.0.1 - - [03/May/2018 12:44:42] "GET /certifi/certifi-2018.1.18-py2.py3-none-any.whl HTTP/1.1" 200 151607
127.0.0.1 - - [03/May/2018 12:44:42] "GET /simple/urllib3/ HTTP/1.1" 200 352
127.0.0.1 - - [03/May/2018 12:44:42] "GET /urllib3/urllib3-1.22-py2.py3-none-any.whl HTTP/1.1" 200 132332
127.0.0.1 - - [03/May/2018 12:44:42] "GET /urllib3/urllib3-1.22.tar.gz HTTP/1.1" 200 226083
127.0.0.1 - - [03/May/2018 12:44:42] "GET /simple/chardet/ HTTP/1.1" 200 356
127.0.0.1 - - [03/May/2018 12:44:42] "GET /chardet/chardet-3.0.4.tar.gz HTTP/1.1" 200 1868453
127.0.0.1 - - [03/May/2018 12:44:42] "GET /chardet/chardet-3.0.4-py2.py3-none-any.whl HTTP/1.1" 200 133356
__________________________ test_case_changes_windows __________________________

PipenvInstance = <class 'tests.integration.conftest._PipenvInstance'>
pypi = <pytest_pypi.serve.Server object at 0x0000028F73D11208>

    @pytest.mark.project
    def test_case_changes_windows(PipenvInstance, pypi):
        """Test project matching for case changes on Windows.
        """
        with PipenvInstance(pypi=pypi, chdir=True) as p:
            c = p.pipenv('install pytz')
            assert c.return_code == 0

            virtualenv_location = Project().virtualenv_location
            target = p.path.upper()
            if target == p.path:
                target = p.path.lower()
            os.chdir('..')
            os.chdir(target)
            assert os.path.abspath(os.curdir) != p.path

            venv = p.pipenv('--venv').out
>           assert venv.strip().lower() == virtualenv_location.lower()
E           AssertionError: assert 'c:\\users\\g...roject\\.venv' == 'c:\\users\\gt...penv-dev-venv'
E             - c:\users\gtalarico\appdata\local\temp\pipenv-h9pdjlw6-project\.venv
E             + c:\users\gtalarico\.env\pipenv-dev-venv

tests\integration\test_windows.py:29: AssertionError
---------------------------- Captured stdout call -----------------------------
$ pipenv install pytz
Installing pytzâ€¦
Looking in indexes: http://127.0.0.1:55733/simple
Collecting pytz
  Downloading http://127.0.0.1:55733/pytz/pytz-2018.3-py2.py3-none-any.whl (509kB)
Installing collected packages: pytz
Successfully installed pytz-2018.3

Adding pytz to Pipfile's [packages]â€¦
Installing dependencies from Pipfile.lock (0afcbd)â€¦
To activate this project's virtualenv, run the following:
 $ pipenv shell

Creating a virtualenv for this projectâ€¦
Using C:\Users\gtalarico\.env\pipenv-dev-venv\Scripts\python.exe (3.6.3) to create virtualenvâ€¦
Already using interpreter C:\Users\gtalarico\.env\pipenv-dev-venv\Scripts\python.exe

Using real prefix 'c:\\python36'

New python executable in C:\Users\gtalarico\AppData\Local\Temp\pipenv-h9pdjlw6-project\.venv\Scripts\python.exe

Installing setuptools, pip, wheel...done.


Virtualenv location: C:\Users\gtalarico\AppData\Local\Temp\pipenv-h9pdjlw6-project\.venv
Creating a Pipfile for this projectâ€¦
Pipfile.lock not found, creatingâ€¦
Locking [dev-packages] dependenciesâ€¦
Locking [packages] dependenciesâ€¦
Updated Pipfile.lock (0afcbd)!

$ pipenv --venv
C:\Users\gtalarico\AppData\Local\Temp\pipenv-h9pdjlw6-project\.venv


---------------------------- Captured stderr call -----------------------------
127.0.0.1 - - [03/May/2018 12:45:01] "GET /simple/pytz/ HTTP/1.1" 200 330
127.0.0.1 - - [03/May/2018 12:45:01] "GET /pytz/pytz-2018.3-py2.py3-none-any.whl HTTP/1.1" 200 509950
127.0.0.1 - - [03/May/2018 12:45:07] "GET /simple/pytz/ HTTP/1.1" 200 330
127.0.0.1 - - [03/May/2018 12:45:07] "GET /simple/pytz/ HTTP/1.1" 200 330
127.0.0.1 - - [03/May/2018 12:45:07] "GET /pytz/pytz-2018.3-py2.py3-none-any.whl HTTP/1.1" 200 509950
127.0.0.1 - - [03/May/2018 12:45:08] "GET /simple/pytz/ HTTP/1.1" 200 330
127.0.0.1 - - [03/May/2018 12:45:08] "GET /pytz/pytz-2018.3-py2.py3-none-any.whl HTTP/1.1" 200 509950
127.0.0.1 - - [03/May/2018 12:45:08] "GET /pytz/pytz-2018.3.tar.gz HTTP/1.1" 200 314214
===Flaky Test Report===


===End Flaky Test Report===
================== 5 failed, 163 deselected in 99.20 seconds ==================
```

--- a/tests/integration/conftest.py
+++ b/tests/integration/conftest.py
@@ -10,6 +10,11 @@ from pipenv.vendor import requests
 from pipenv.vendor import six
 from pipenv.vendor import toml
 
+try:
+    from pathlib import Path
+except ImportError:
+    from pipenv.vendor.pathlib2 import Path
+
 
 if six.PY2:
     class ResourceWarning(Warning):
@@ -45,7 +50,7 @@ class _PipenvInstance(object):
         self.original_umask = os.umask(0o007)
         self.original_dir = os.path.abspath(os.curdir)
         self._path = TemporaryDirectory(suffix='-project', prefix='pipenv-')
-        self.path = self._path.name
+        self.path = str(Path(self._path.name).resolve())
         # set file creation perms
         self.pipfile_path = None
         self.chdir = chdir

